frontend/lib/util/session-predicates.ts

```diff
@@ -4,3 +4,8 @@ import { AllSessionInfo } from "../queries/AllSessionInfo";
 export function isUserLoggedIn(s: AllSessionInfo): boolean {
   return !!s.phoneNumber;
 }
```

Comment:
Ooooh! This is the first time I've truly noticed this "double bang" js syntax— super helpful to keep in mind for returning truthy as a boolean :)

Reply:
Ohhh yeah it is super useful although a bit annoying to read.  I keep have to mentally walk through the logic every time I see it.